app/assets/src/components/views/compare/SamplesHeatmapView.jsx

```diff
@@ -715,12 +725,21 @@ class SamplesHeatmapView extends React.Component {
                     text="Share"
                     onClick={this.onShareClick}
                     className={cs.controlElement}
+                    icon={
+                      <Icon size="large" className={"cloud share alternate"} />
+                    }
                   />
                 }
                 content="A shareable URL has been copied to your clipboard!"
                 on="click"
                 hideOnScroll
               />
+              <SecondaryButton
```

Comment:
Since we are going to have multiple of these buttons, can you create a `SaveButton` and `ShareButton` to encapsulate these?

Reply:
Seconded, especially given we’re going to have to change the look of these buttons in the upcoming design change